### PR TITLE
Add card component

### PR DESCRIPTION
**Pull request by haeminn**
## Summary

Figma: https://www.figma.com/file/FVaCxUjkCXhIijhndlOnGH/Orchestration-~-Build-Specifications?node-id=10810%3A219408

## Checklist

- [x] \*Added unit tests
- [x] \*Added a code reviewer
- [ ] Added changes to `/projects/swimlane/ngx-ui/CHANGELOG.md` under HEAD (Unreleased)
- [x] Updated the demo page
- [ ] Included screenshots of visual changes

_\*required_

### CODE REVIEW

File: src/app/components/card-page/card-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    }
+
+    .card-action {
+      justify-content: space-between;
+      padding-right: 0 !important;
+      color: $color-blue-grey-350;
+
+      .ellipsis {
+        margin-right: 12px;
+        padding: 0;
```
Comment (nelsonGuti) [inline src/app/components/card-page/card-page.component.scss:42]: Not sure if this is causing it, but it seems the dropdown menu toggled from the ellipsis is slightly misaligned

image

File: projects/swimlane/ngx-ui/src/lib/components/card/card.component.scss [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+      .ngx-card-tag {
+        font-weight: bold;
+        font-size: 10px;
+        line-height: 12px;
+      }
+
+      .ngx-card-title {
+        font-weight: 600;
+        font-size: 22px;
+        color: #f0f1f6;
```
Comment (nelsonGuti) [inline projects/swimlane/ngx-ui/src/lib/components/card/card.component.scss:130]: Question more than an actual comment, do we want to ellipse on long names on vertical card?
image

Comment (nartc) [reply projects/swimlane/ngx-ui/src/lib/components/card/card.component.scss:130]: issue: it seems like a fixed height on the card itself (418px) is causing the issue with the footer as shown in Nelson's screenshot with long titles as well

File: src/app/components/card-page/card-page.component.scss (hunk 1 above)
Comment (haeminn) [reply src/app/components/card-page/card-page.component.scss:42]: I think someone mentioned issues with the dropdown caret not aligning with dropdown toggle before. @Hypercubed Do you remember?

File: projects/swimlane/ngx-ui/src/lib/components/card/card.component.scss (hunk 2 above)
Comment (haeminn) [reply projects/swimlane/ngx-ui/src/lib/components/card/card.component.scss:130]: Yeah - missed this from design. Long text needs ellipsis.

**Review by Hypercubed — APPROVED** [review]

**Review by mcwenger — CHANGES_REQUESTED** [review]
1. There's a margin issue on the placeholder last-child that shifts it visually off center. Can we add `:last-child: margin: 0` to `ngx-card-placeholder`.
2. The description needs a flex-basis width of 28% so when in a sequential list they align, and don't take over the row.
3. Can we limit descriptions to 3 lines? I.e. `line-clamp: 3` or however you want to do that.

Thanks!

**Review by mcwenger — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**